py/DREAM/Settings/Equations/Ions.py

```diff
@@ -311,6 +311,17 @@ def setIonType(self, index, ttype):
             raise DREAMException("Trying to set invalid ion type for ion species '{}': {}.".format(ion.name, ttype))
 
         ion.ttype = ttype
+
+        
+    def shiftTimeTranspCoeffs(self, tShift):
```

Comment:
Please add a comment to this method describing what the method is supposed to do.

Reply:
It is similar as with the ion densities, you specify the transport coefficients separately for each species, but the grids on which you specify them must be the same for all species.